crates/credentials/src/vc.rs

```diff
@@ -6,6 +6,8 @@ use jwt::{
     {Claims, JwtError, RegisteredClaims},
 };
 use serde::{Deserialize, Serialize};
+use std::cmp::PartialEq;
```

Comment:
fun fact, you don't have to `use` this because it's a part of the "rust prelude" which means it's automatically imported